test/testcafe/spec/uiDemo_spec.js

```diff
@@ -0,0 +1,42 @@
+import { RequestMock } from 'testcafe';
+import UiDemoPageObject from '../framework/page-objects/UiDemoPageObject';
+import uiDemoResponse from '../../../playground/mocks/data/idp/idx/_ui-demo.json';
+
+import compareScreenshot from '../../../vrtUtil/vrtUtil';
+
+const uiDemoMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(uiDemoResponse);
+
+async function setup(t) {
+  const pageObject = new UiDemoPageObject(t);
+  await pageObject.navigateToPage();
+  await t.expect(pageObject.formExists()).eql(true);
+  return pageObject;
+}
+
+fixture('UI demo').meta('gen3', false).meta('gen2', false);
```

Comment:
this spec runs in its own bacon suite and is excluded from running in the gen 2 `testcafe` and `parity-v3` run